### ISSUE TITLE
Mixins on server rendering

### ISSUE THREAD

**Issue by wesleycoder** [rofrischmann/react-look#274]
**Version**: 1.0.0 (beta8)
**Environment**: React
**Type**: Bug

-------------------
#### Description
Server rendering does not output the styles of extend mixins.

#### Steps to reproduce
1. Use server rendering with `StyleSheet.renderToString`
2. Use `react-dom` preset
3. Use `extend` mixin on StyleSheet

#### Expected Behavior
Should render classNames with "extended" classes like "c6 c6-d-72geds";

#### Actual Behavior
Render just the base classNames like "c6";

#### Obs:
If needed I can share my code to help track down the cause.

**Comment by rofrischmann** [comment]
Does this only happen with server-side rendering? Or is there a general issue that prevents extend of working at all?

**Comment by wesleycoder** [comment]
Just on server-side. This throws the react warnings about the diff on the generated markup.
```
Warning: React attempted to reuse markup in a container but the checksum was invalid.
This generally means that you are using server rendering and the markup generated on
the server was not what the client was expecting.
React injected new markup to compensate which works but you have lost many of the
benefits of server rendering.
Instead, figure out why the markup being generated is different on the client or server:
 (client) id="11"><a class="c6 c6-d-72geds" href="
 (server) id="11"><a class="c6" href="/" data-reac
```

**Comment by wesleycoder** [comment]
Maybe this is related to #276 ?

**Comment by rofrischmann** [comment]
Yeah I was thinking the same. We might close this and focus on the other one as it should fix this one too if that's ok for you. The problem seems to be caused by the same problem.
